docs/index.rst

```diff
@@ -3,14 +3,27 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Switchboard's documentation!
-=======================================
+Welcome to Switchboard
+======================
 
-Contents:
+Switchboard is a feature flipper library for the Pyramid or Pylons stack
+(including TurboGears). Originally used to selectively roll out changes to the
+SourceForge site, the library lets you easily control whether a particular
+change (a switch) is active.
+
+You can make switches active for a certain percentage of visitors, all
+visitors to a particular host in a cluster, or if a particular string is
+present in the query string. Furthermore you can easily create your own
+conditions to do fancier things like geo-targeting, specific users, etc.
+In short, Switchboard turns you into a continuous deployment ninja.
```

Comment:
Au contraire: from Wikipedia's [Feature Toggle](https://en.wikipedia.org/wiki/Feature_toggle) article:

> Continuous release and continuous deployment provide developers with rapid feedback about their coding. This requires the integration of their code changes as early as possible. Feature branches introduce a bypass to this process. Feature toggles bring developers back to the track, but the execution paths of their features are still "dead" and "untested" if a toggle is "off". But the effort is low to enable the new execution paths just by setting a toggle to "on".

I'll update the docs to make the connection more clear.